includes/models/model.llms.student.php

```diff
@@ -1093,7 +1125,7 @@ public function get_registration_date( $format = '' ) {
 	 *
 	 * @since 3.14.0
 	 *
-	 * @return boolean
+	 * @return bool
```

Comment:
You changed all the occurrences of `@return boolean` in this file?
Thanks for this (really mean it), but please next time do it in a different PR, as low priority task.
If you do this kind of changes in a big PR like this it only makes the review more difficult. The rule of thumb is that we update the documentation or do small refactors only of the code which is "around" the portion of code we need to change.
E.g. it's ok fixing the docblock and coding standards of a specific method if we're already acting on that method to add a new feature or fix a bug. It's not convenient to fix the docblock and coding standard of a sibling method, or of a totally different  file, it's just harder to review.